astroquery/casda/tests/test_casda.py

```diff
@@ -274,10 +272,8 @@ def test_query_region_async_box(patch_get):
 
 
 def test_filter_out_unreleased():
-    # The ``W06: Invalid UCD 'meta.ref.url;meta.curation'`` warning is only raised with older astropy
-    with pytest.warns(W06) if ASTROPY_LT_5_3 else nullcontext():
```

Comment:
I'm not sure why this disappeared with updating the minimum pytest version, but it's gone, and as it was a workaround anyway I don't investigate further.